demo/react/doc/product/foundations/iconography.mdx

```diff
@@ -0,0 +1,35 @@
+```javascript import
+import { Icon, Size, Theme } from 'LumX';
+import { mdiEmail } from 'LumX/icons';
+```
+
+
+# Iconography
+
+**Icons are small visual representations of common actions, objects or ideas. They help users to identify information and provide context.**
+
+## Sizes
+Comes in 5 sizes: xs, s, m, l, xl.
+```javascript jsx withThemeSwitcher
+(theme) => (
+    <>
+        <Icon icon={mdiEmail} theme={theme} size={Size.xs}>
+            Header with icon
```

Comment:
The Icon component doesn't have content.
You can remove the text `Header with icon` and the closing tag `</Icon>`:
```jsx
<Icon icon={mdiEmail} theme={theme} size={Size.xs} />
```

Reply:
Done ;)